### DISCUSSION TITLE
Compressing lines of comments similar to compressing lines of functions

### DISCUSSION THREAD

**Discussion by pylotpig** [godotengine/godot-proposals#5988] [Editor]
I just noticed after using other IDES such as IntelliJ that we can't compress multiple lines of comments in godot. But we can do it for functions. It would be a quality of life update if this was built in and implemented.
<img width="587" alt="Screenshot 2022-12-24 103628" src="https://user-images.githubusercontent.com/121233523/209430356-60a5487f-6bb9-4e62-a3e7-3d9acb6157b8.png">
<img width="288" alt="Screenshot 2022-12-24 103813" src="https://user-images.githubusercontent.com/121233523/209430361-aba088ce-c723-4ef3-bd3d-9c22f67391ea.png">
<img width="207" alt="Screenshot 2022-12-24 103824" src="https://user-images.githubusercontent.com/121233523/209430365-72124bac-c1b5-439c-b32a-6d1ce80372cf.png">

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot-proposals/issues/3366.

**Comment by Mickeon** [comment]
Multiple comment lines can be compressed down in Godot 4 if I recall correctly.
